source/sass/buyers-guide/views/homepage.scss

```diff
@@ -1,10 +1,12 @@
 #view-home {
+  $breakpoint-larger: $bp-md;
```

Comment:
probably here too, "larger" is being set to "medium", we probably want a clearer name